llvm/docs/DeveloperPolicy.rst

```diff
@@ -293,57 +293,71 @@ warranted when performing a code review.
 Quality
 -------
 
-The minimum quality standards that any change must satisfy before being
-committed to the main development branch are:
+Sending patches, PRs, RFCs, comments, etc to LLVM, is not free -- it takes a lot
+of maintainer time and energy to review those contributions! Our **golden rule**
+is that a contribution should be worth more to the project than the time it
+takes to review it. These ideas are captured by this quote from the book
+`Working in Public <https://press.stripe.com/working-in-public>`_ by Nadia
+Eghbal:
+
+  .. pull-quote::
+
+    "When attention is being appropriated, producers need to weigh the costs and
+    benefits of the transaction. To assess whether the appropriation of attention
+    is net-positive, it’s useful to distinguish between *extractive* and
+    *non-extractive* contributions. Extractive contributions are those where the
+    marginal cost of reviewing and merging that contribution is greater than the
+    marginal benefit to the project’s producers. In the case of a code
+    contribution, it might be a pull request that’s too complex or unwieldy to
+    review, given the potential upside."
+    -- Nadia Eghbal
+
+We encourage non-extractive contributions that help sustain the project. We want
+the LLVM project to be welcoming and open to aspiring compiler engineers who are
+willing to invest time and effort to learn and grow, because growing our
+contributor base and recruiting new maintainers helps sustain the project over
+the long term. We therefore label pull requests from new contributors and
```

Comment:
We do? What's the label for that?

Reply:
I based this on glancing at this github workflow:
https://github.com/llvm/llvm-project/blob/main/.github/workflows/new-prs.yml#L24

On second glance, we don't have a label, but we do post a greeting. I'll just update the text to match reality, but I think a label would be helpful too, since it can be used to specifically focus on reviewing PRs from new contributors. I realize I'm speaking as someone who only reviews things sent to me, but this seems like a good aspirational goal.